surveys/2022-wg-debugging-survey/README.md

```diff
@@ -0,0 +1,3 @@
+# `t-compiler/wg-debugging` survey
+The compiler team's new debugging working group would like to survey users to
+better understand their experiences debugging Rust.
```

Comment:
It would be interesting to know the high-level goals of this survey. What high-level questions are we trying to answer? Knowing this makes it easier to tell if the specific questions we're asking will help us answer the high-level information we're really after.